test/shared/src/main/scala-2.x/zio/test/AssertionVariants.scala

```diff
@@ -23,11 +23,13 @@ trait AssertionVariants {
   /**
    * Makes a new assertion that requires a value equal the specified value.
    */
-  final def equalTo[A, B](expected: A)(implicit eql: Eql[A, B]): Assertion[B] =
-    Assertion.assertion("equalTo")(param(expected)) { actual =>
+  final def equalTo[A, B](expected: A)(implicit eql: Eql[A, B]): Assertion[B] = {
```

Comment:
We should make sure we replicate any changes here across the other version specific file.